google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -268,6 +268,29 @@ def encryption_key_sha256
         end
 
         ##
+        # The file's storage class. This defines how the file is stored and
+        # determines the SLA and the cost of storage.
+        def storage_class
+          @gapi.storage_class
+        end
+
+        ##
+        # Updates how the file is stored and determines the SLA and the cost of
+        # storage. Values include `:multi_regional`, `:regional`, `:nearline`,
+        # `:coldline`, `:standard`, and `:dra` (Durable Reduced Availability),
+        # as well as the strings returned by {File#storage_class} or
+        # {Bucket#storage_class}. For more information, see [Per-Object Storage
+        # Class](https://cloud.google.com/storage/docs/per-object-storage-class).
```

Comment:
If only one official docs page is linked, I think [docs/storage-classes](https://cloud.google.com/storage/docs/storage-classes) is the better one to clarify the choices. The `docs/per-object-storage-class` page linked here is also valuable, especially for the region vs multi-region section, but less relevant.